lib/interface/cli/commands/pipeline/run.cmd.js

```diff
@@ -86,6 +86,11 @@ const run = new Command({
                 default: [],
                 alias: 'v',
             })
+            .option('secret', {
+                describe: 'Set build secrets',
+                default: [],
+                alias: 'sec',
```

Comment:
Could we check if it's free?
```suggestion
                alias: 'S',
```